doc/dev_guide/internals/file_formats.rst

```diff
@@ -10,9 +10,8 @@ Data persistence and the WAL file format
 
 To maintain data persistence, Tarantool writes each data change request (insert,
 update, delete, replace, upsert) into a write-ahead log (WAL) file in the
-:ref:`wal_dir <cfg_basic-wal_dir>` directory. A new WAL file is created for every
-:ref:`rows_per_wal <cfg_binary_logging_snapshots-rows_per_wal>` records, or for every
-:ref:`wal_max_size <cfg_binary_logging_snapshots-wal_max_size>` bytes.
+:ref:`wal_dir <cfg_basic-wal_dir>` directory. A new WAL file is created
+when :ref:`wal_max_size <cfg_binary_logging_snapshots-wal_max_size>` is reached.
```

Comment:
It's not completely clear _what_ reaches this limit. Maybe smth like `when the current one reaches ...`

Reply:
thanks, sounds much clearer